test/shutdown-testing.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -eu
+
+declare appPID
+
+_get_app_pid() {
+  appPID=($(jps | grep ApplicationKt))
+}
+
+_kill_spring_app() {
+  kill -SIGINT ${appPID}
+}
+
+main(){
```

Comment:
You can inline these statements, and declare appPID as a local. If it ever gets more complex, we can break them out into functions, but it's just one command per function.